### DISCUSSION TITLE
[✨] Make it easier to get a hold of a manifest

### DISCUSSION THREAD

**Discussion by mhevery** [QwikDev/qwik-evolution#148] [Proposals For Qwik]
### Is your feature request related to a problem?

Let's say I want to call `rederToString` internally inside the application (For example, to send an email.) The renderToString needs a manifest; otherwise, it fails to render. There is no easy way to get a hold of the manifest in an existing application. (Specifically inside a `RouteHandler`)

### Describe the solution you'd like

Some way to get a hold of the manifest inside the `RouteHandler` `routeLoader$()` etc...

### Describe alternatives you've considered

importing 
```
import { manifest } from '@qwik-client-manifest';
```

Does not work in dev mode because there is no manifest. In such a case it would be nice to get ahold of the `symbolMapper`

**Comment by manucorporat** [comment]
I have something in mind for this, it will be required for `worker$()`, and "client-side pausing"

**Comment by rqzheng2015** [comment]
@mhevery @manucorporat Hi, guys, any progress here? We met the same issues when we want to build a SSR service but we noticed that manifest is only generated when `opts.target === 'client'`, which doesn't meet our need.
Thus, we also want to get a more easier way to generate the manifest and get it running on server, which might be capable for migration of qwik to express server without building qwik application from scratch.  
Please have a look, thanks.

**Comment by gioboa** [comment]
We moved this issue to `qwik-evolution` repo to create a RFC discussion for this.
Here is our [Qwik RFC process](https://github.com/QwikDev/qwik-evolution?tab=readme-ov-file#qwik-rfc-process) thanks.

**Comment by DmitryEfimenko** [comment]
it looks like this was moved out of the issue back to discussion? I can't see this in [projects](https://github.com/orgs/QwikDev/projects/4/views/1?filterQuery=manifest)

**Comment by wmertens** [comment]
This is no longer an issue, dev mode doesn't need the symbolmapper any more, and rendering automatically grabs the manifest.
